src/Tasks/Microsoft.NET.Build.Tasks/msbuildExtensions-ver/Microsoft.Common.targets/ImportAfter/Microsoft.NET.Build.Extensions.targets

```diff
@@ -0,0 +1,7 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MicrosoftNETBuildExtensionsTargets Condition="'$(MicrosoftNETBuildExtensionsTargets)' == ''">$(MSBuildExtensionsPath)\Microsoft.NET.Build.Extensions\Microsoft.NET.Build.Extensions.targets</MicrosoftNETBuildExtensionsTargets>
```

Comment:
I am concerned about the risk of adding targets to the build of all 15.3 .NET Framework projects (including classic csproj) at this stage of the 15.3 cycle.

* Have we adequately measured the performance impact given that this can impact all .NET framework projects, and hence a much larger set of customer projects.

* Do we have the approval that will be required to put this additional feature in-box in 15.3? I don't think it meets the bar that @MattGertz sent out recently without a special exception. 

I'm also unhappy with the complexity this adds to risk assessment of code changes in this repo moving forward. Previously, we could say that any change in the release/2.0.0 branch was scoped to the out-of-band .NET Core 2.0 SDK install, but now these targets are destined to go in-box, and so we'd have to audit each change to see the true impact and to know what level of approval we need. Furthermore, with the shared source files, it's not easy to see what changes are headed in-box and what changes are not. :(

Previously, insertions from CLI -> VS carried only 1.1 CLI and 2.0 resolver. This was simple to reason about. With this, we'd also have a chunk of the 2.0 SDK code going in to that payload. I want to be absolutely sure that the important decision makers that pushed to limit the churn from .NET Core 2.0 to 15.3 are fully aware of this before we move forward with it.